source/common.ts

```diff
@@ -141,6 +144,21 @@ const i18n = {
 
     return translationWithHideMissing;
   },
+  _getPluralization(translation: string, locale: string, count?: number) {
+    const pluralizationRules = _i18n.options.pluralizationRules;
+    if (count !== undefined && typeof translation === 'string') {
```

Comment:
This condition is weird, since `translation: string`. But I guess it's `string | Record<string, unknown> | undefined` actually.